pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -157,6 +157,22 @@ type PodInfo struct {
 	Pod *v1.Pod
 	// The time pod added to the scheduling queue.
 	Timestamp time.Time
+	// Number of schedule attempts before successfully scheduled.
```

Comment:
can you please update the comment for PodInfo at line 155, not sure what "cell" means. Can you please indicate that all those members lnes 159-165 are for instrumentation.